app/src/main/res/values/styles.xml

```diff
@@ -8,6 +8,9 @@
         <item name="android:windowLightStatusBar" tools:targetApi="23">false</item>
         <item name="android:statusBarColor">@color/colorPrimaryDark</item>
         <item name="android:windowDrawsSystemBarBackgrounds">true</item>
+        <item name="android:windowTranslucentStatus">true</item>
```

Comment:
For now you can create a new sub Theme because with this you changed the UI of the whole application xD